service/responses.tsp

```diff
@@ -0,0 +1,152 @@
+import "@typespec/http";
+import "@typespec/openapi";
+
+using Http;
+
+@tag("Responses")
+namespace CoreSystem.Responses {
+  @doc("The available answer types")
+  enum AnswerType {
+    shortText: "short_text",
+    longText: "long_text",
+    singleChoice: "single_choice",
+    multipleChoice: "multiple_choice",
+    date: "date",
+  }
+
+  @doc("A form response submission")
+  model Response {
+    @doc("The response's unique identifier.")
+    id: uuid;
+
+    @doc("The form that this response belongs to.")
+    formId: uuid;
+
+    @doc("The user who submitted this response.")
+    submittedBy: uuid;
+
+    @doc("The creation timestamp of the response.")
+    createdAt: utcDateTime;
+
+    @doc("The last updated timestamp of the response.")
+    updatedAt: utcDateTime;
+  }
+
+  @doc("An individual answer")
+  model Answer {
+    @doc("The answer's unique identifier.")
+    id: uuid;
+
+    @doc("The response this answer belongs to.")
+    responseId: uuid;
+
+    @doc("The question this answer is for.")
+    questionId: uuid;
+
+    @doc("The type of answer, determines how to parse answerValue.")
+    answerType: AnswerType;
+
+    @doc("The answer value stored as string, parsed based on answerType.")
+    answerValue: string;
+
+    @doc("The creation timestamp of the answer.")
+    createdAt: utcDateTime;
+
+    @doc("The last updated timestamp of the answer.")
+    updatedAt: utcDateTime;
+  }
+
+  @doc("Request model for submitting an individual answer")
+  model AnswerRequest {
+    @doc("The question being answered.")
+    questionId: uuid;
+
+    @doc("The type of answer.")
+    answerType: AnswerType;
+
+    @doc("The answer value (format depends on answerType).")
+    answerValue: string;
+  }
+
+  @doc("Request model for submitting a complete form response")
+  model SubmitFormResponseRequest {
+    @doc("All answers for this form submission.")
+    answers: AnswerRequest[];
+
+    @doc("User ID for authenticated submissions.")
+    submittedBy: uuid;
```

Comment:
If `submittedBy` is meant to represent the authenticated user submitting the request, then accepting it from the request body introduces a security risk. It should be extracted from the request context (e.g., via middleware), not trusted from external input.